addons/clones/spriteCount.js

```diff
@@ -0,0 +1,68 @@
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+  const ogCloneCounter = vm.runtime.changeCloneCounter;
+
+  function updateCounts(remove) {
+    let counts = {};
+
+    vm.runtime.targets
+      .filter((target) => !target.isOriginal)
+      .map((target) => target.sprite.name)
+      .forEach((target) => {
+        if (!counts[target]) counts[target] = 1;
+        else counts[target] += 1;
+      });
+
+    const spriteNames = Array.from(
+      document
+        .querySelector("[class*=sprite-selector_items-wrapper]")
+        .querySelectorAll("[class*=sprite-selector-item_sprite-name]")
+    );
+
+    spriteNames.forEach((spriteName) => {
+      spriteName.querySelector(".clone-count")?.remove();
+      if ((counts[spriteName.innerText.split("\n")[0]] !== undefined) & !remove) {
+        const count = document.createElement("div");
+
+        count.classList.add("clone-count");
+        count.innerText = `(${counts[spriteName.innerText.split("\n")[0]]} clones)`;
+        spriteName.appendChild(count);
+      }
+    });
+  }
```

Comment:
maybe there is a different method (similar to how the stage clone count already works) to not have to delete and create an element everytime.

Reply:
I could make it not delete and re-add for things like just updating the count but whenever a sprite rename occurs it does need to be readded. I'll look into it.

Reply:
Doneeeeee